install.sh

```diff
@@ -83,6 +83,16 @@ configure() {
     args=""
     add_arg() {
         log_debug "Add arg -$1=$2"
+        multiline=$(is_multiline "$2")
+        if [ "$multiline" -eq "1" ]; then
+            log_debug "Multiline response detected for arg: ${1}; will iterate"
+            for item in $2
+            do
+                args="$args -$1=$item"
+            done
+        else
+            args="$args -$1=$2"
+        fi
         args="$args -$1=$2"
```

Comment:
Isn't this line a duplicate?

Reply:
actually you can skip the test "is_multiline" and always go for the for loop. It will work the same with a single line as with multiple line.